app/views/benefits/accommodation/LivingAccommodationBenefitsAmountView.scala.html

```diff
@@ -42,9 +42,7 @@
 
     @heading(headingForUse, Some(messages("benefits.caption", (taxYear - 1).toString, taxYear.toString)), "govuk-!-margin-bottom-2")
 
-    @if(preAmount.isDefined) {
-        <p class="govuk-body">@messages("common.ifItWasNot", bigDecimalCurrency(preAmount.get.toString))</p>
-    }
+
```

Comment:
Related to what we mentioned in the controller. As we don't use the `preAmount`, we can just remove the variable? It is not used anywhere else in the file as far as we can see.

Reply:
I will fix it.